Mathlib/Logic/IsEmpty.lean

```diff
@@ -233,3 +233,10 @@ theorem biTotal_iff_isEmpty_right [IsEmpty α] : BiTotal R ↔ IsEmpty β := by
 
 theorem biTotal_iff_isEmpty_left [IsEmpty β] : BiTotal R ↔ IsEmpty α := by
   simp only [BiTotal, leftTotal_iff_isEmpty_left, rightTotal_empty, and_true]
+
+lemma Function.Injective.of_isEmpty (f : α → β) [IsEmpty α] : f.Injective := IsEmpty.elim ‹_›
+
+lemma Function.Surjective.of_isEmpty (f : α → β) [IsEmpty β] : f.Surjective := IsEmpty.elim ‹_›
+
+lemma Function.Bijective.of_isEmpty (f : α → β) [IsEmpty α] [IsEmpty β] : f.Bijective :=
```

Comment:
You can remove `[IsEmpty α]` using `Function.isEmpty`, and then the separate surjectivity theorem might be redundant